tests/test_classification.py

```diff
@@ -194,3 +195,149 @@ def test_TSclassifier_classifier_error():
     """Test TS if not Classifier"""
     with pytest.raises(TypeError):
         TSclassifier(clf=Covariances())
+
+
+def test_Quantic_init():
```

Comment:
This probably could be improved. Preferably in another PR as this one is already big enough :)